source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/admin/settings/organizations/edit/index.tsx

```diff
@@ -0,0 +1,93 @@
+import * as React from 'react';
+import * as Toast from '@lib/toast';
+import { compose } from 'recompose';
+import { withRouter, RouteComponentProps } from 'react-router-dom';
+import { withData as withOrbit } from 'react-orbitjs';
+
+import {
+  withDataActions,
+  IProvidedProps as IOrganizationProps
+} from '@data/containers/resources/organization/with-data-actions';
+
+import OrganizationForm from '../common/form';
+
+import { listPathName } from '../index';
+import { query, withLoader, buildOptions, buildFindRecord, OrganizationResource, UserResource } from '@data';
+
+interface IOwnProps {
+  organization: OrganizationResource;
+  owner: UserResource;
+}
+
+type IProps =
+  & IOrganizationProps
+  & IOwnProps
+  & RouteComponentProps<{}>;
+
+class EditOrganization extends React.Component<IProps> {
+
+  update = async (attributes, onSuccess) => {
+
+    const { updateAttributes, updateOwner } = this.props;
+
+    try {
+      await updateAttributes({
+        name: attributes.name,
+        websiteUrl: attributes.websiteUrl,
+        buildEngineUrl: attributes.buildEngineUrl,
+        buildEngineApiAccessToken: attributes.buildEngineApiAccessToken,
+        logoUrl: attributes.logoUrl,
+        publicByDefault: attributes.publicByDefault
+      });
+      await updateOwner(attributes.owner);
+      onSuccess();
+      this.redirectToList();
+      Toast.success('Organization updated');
```

Comment:
why is this capital?

Reply:
Just because I imported using capital, because its a collection of functions. But I can change it if you like

Reply:
Only classes are PascalCase, afaik

Reply:
changed